src/Kernel/Magic/PackageMagic.cs

```diff
@@ -26,7 +26,48 @@ public class PackageMagic : AbstractMagic
         public PackageMagic(IReferences references) : base(
             "package",
             new Documentation {
-                Summary = "Provides the ability to load a Nuget package. The package must be available on the list of nuget sources, typically this includes nuget.org"
+                Summary = "Provides the ability to load a NuGet package.",
+                Description = @"
+                    This magic command allows for loading a NuGet package into the current IQ# kernel process.
+                    The package must be available on the system's list of NuGet sources, which typically includes nuget.org.
+                    Functionality such as magic commands and result encoders defined in the loaded package will
```

Comment:
It may be helpful to note here that Q# operations, functions, and UDTs are also made available.

Reply:
I guess that's the important part, isn't it? 😊 I've been staring at too many magic commands.